packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -126,9 +128,8 @@
 }
 
 .main-content {
-  @apply grid grid-cols-[calc(100vw_-_1.5rem)] grid-rows-[min-content] gap-x-4 mt-16 min-h-[calc(100vh_-_4rem)];
-  @apply tablet:grid-cols-[minmax(min-content,500px)_minmax(20rem,1fr)] tablet:gap-x-8 tablet:mt-[80px] tablet:mb-auto tablet:mx-3;
-  @apply desktop:grid-cols-[600px_1fr];
```

Comment:
`@apply desktop:grid-cols-[600px_1fr];` caused this weird gap that was fixed by applying `tablet:h-0` in some views (e.g. mobile and desktop landscape). 

However, removing this line and `tablet:h-0` fixes the issue in desktop, and mobile landscape has the same layout as mobile portrait.
<img width="1045" alt="Screenshot 2022-11-21 at 7 53 06 PM" src="https://user-images.githubusercontent.com/28129806/203187985-5483401e-fe80-4758-96b4-bba6ecbedf40.png">

Reply:
Looks like without the `@apply desktop:grid-cols-[600px_1fr];` the left column card padding snaps to less padding, before the card starts shrinking. At least with the grid cols set to 600px, it snaps the card to 500px and lowers the padding.

Should we add the `desktop` line back in or what do you think?